hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/PrisonersInPrisonResource.kt

```diff
@@ -127,20 +134,34 @@ class PrisonersInPrisonResource(private val searchService: PrisonersInPrisonServ
     @RequestParam(value = "incentiveLevelCode", required = false)
     @Parameter(description = "Filter for the prisoners on an incentive level.", example = "STD")
     incentiveLevelCode: String?,
+    @RequestParam(value = "requestedAttributes", required = false)
+    @Parameter(description = "A list of attributes to return on the Prisoner record. An empty list defaults to all attributes.", example = "prisonerNumber,firstName")
+    requestedAttributes: List<String>? = null,
```

Comment:
The main change is to accept a list of attributes to populate on the returned `Prisoner` record.